client/account-recovery/reset-password/reset-password-form/index.jsx

```diff
@@ -30,15 +33,52 @@ export class ResetPasswordFormComponent extends Component {
 	};
 
 	submitForm = () => {
+		// TODO:
+		// This is going to be replaced by corresponding redux actions.
 		this.setState( { isSubmitting: true } );
 	};
 
 	onResetOptionChanged = ( event ) => {
 		this.setState( { selectedResetOption: event.currentTarget.value } );
 	};
 
-	render() {
+	getOptionDisplayStrings = ( optionName ) => {
 		const { translate } = this.props;
+
+		switch ( optionName ) {
+			case 'primary':
+				return {
+					email: translate(
+						'Email a reset link to {{strong}}your main email address{{/strong}}',
+						{ components: { strong: <strong /> } }
+					),
+					sms: translate(
+						'Send a reset code to {{strong}}your main phone{{/strong}}',
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Send a reset code to {{strong}}your phone{{/strong}}' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).